bin/exec-env

```diff
@@ -1,9 +1,6 @@
 #!/usr/bin/env bash
 
-plugin_name="luaJIT"
-
 install_path="$(asdf where luajit)"
-full_version="$( echo $install_path | rev | cut -d/ -f1 | rev)"
```

Comment:
`shellcheck` caught that this variable wasn't used.

Reply:
nice